TourMate/TourMate/Backend/Persistence/Config/FirebaseConfig.swift

```diff
@@ -11,7 +11,6 @@ final class FirebaseConfig {
     static let userCollectionId = "users"
     static let tripCollectionId = "trips"
     static let planCollectionId = "plans"
-    static let planDetailsCollectionId = "plandetails"
```

Comment:
Not sure if this is needed. I recall someone asked me to delete.

Reply:
yup not needed